docs/mine-hnt/validators/validators.mdx

```diff
@@ -6,61 +6,88 @@ sidebar_label: Validators (Testnet)
 slug: /mine-hnt/validators
 ---
 
-
 import useBaseUrl from "@docusaurus/useBaseUrl";
 
 <img className="docsheader" src={useBaseUrl("img/mine-hnt/validators.png")} />
 
 # Validators (Testnet)
 
-Validators are a new entity on the Helium Blockchain that will perform the work of the consensus group including verifying transactions and adding new blocks to the blockchain. 
+Validators are a new entity on the Helium Blockchain that will perform the work of the consensus group including verifying transactions and adding new blocks to the blockchain.
 
-<img className="docsheader" style={{marginBottom: 30}} src={useBaseUrl("img/validators/schematic.png")} />
+<img
+  className="docsheader"
+  style={{ marginBottom: 20 }}
+  src={useBaseUrl("img/validators/schematic.png")}
+/>
 
+Before you start, there are a few items to review...
 
-## Before you Start...
+### Expectations for Running a Validator on Testnet
 
-:::info
+As Validators will play an important role in the expansion, stability and success of the Helium network moving forward, it's important to set expectations for running a Validator, especially during the Testnet process.
 
-Use the ```#validator``` channel on our [Discord](https://discord.gg/helium) server to take part in the real-time Validator development, provide feedback, and report issues.
+In participating in Testnet, the expectations are that you will:
 
-:::
+- Join the [Helium Discord](https://discord.gg/helium) server to take part in the real-time Validator development. Participate in the `#validator` channel, but more importantly subscribe to alerts in `#validators-announcements`. `#validators-announcements` will provide you with important, timely information for upgrades and feedback during this process.
+- Be attentive to upgrade the Validator software when requested through `#validators-announcements`. This may include tasks that will require you to delete & re-stake your Validator instance, or perform other tasks that will assist the team in bringing the best product to the network.
+- Be proficient in command-line driven linux-based operating systems, comfortable working with cloud based service providers (AWS, Azure, Linode, DigitalOcean, or GCP, etc.), comfortable with working with firewall rules / NAT and other basic networking components, and experienced in working with docker images and controls.
 
-:::warning 
+The Helium team as well as the greater Helium community is continually attemping to make this process as clear and easy to understand as possible. However, during this time you can expect much change.
 
-During the Testnet, expect frequent updates which could result in API and Faucet service interruptions requiring node restarts.
+:::warning
 
-:::
+If you do not feel you can meet these expectations, thats fine! It may be better to wait for the public availablity of validators. There will be future announcements on the availability of staking your own validator on mainnet.
```

Comment:
```suggestion
We understand that not everyone will be able to meet these expectations. It may be better to wait for the public availability of validators. There will be future announcements on the availability of staking your own validator on mainnet.
```

Softer tone in the first sentence and a typo fix.